mteb/tasks/Clustering/jpn/LivedoorNewsClustering.py

```diff
@@ -2,9 +2,55 @@
 from mteb.abstasks.TaskMetadata import TaskMetadata
 
 
+class LivedoorNewsClusteringv2(AbsTaskClusteringFast):
+    max_document_to_embed = 1107
+    max_fraction_of_documents_to_embed = None
+
+    metadata = TaskMetadata(
+        name="LivedoorNewsClustering.v2",
+        description="Clustering of the news reports of a Japanese news site, Livedoor News by RONDHUIT Co, Ltd. in 2012. It contains over 7,000 news report texts across 9 categories (topics). Version 2 updated on LivedoorNewsClustering by removing pairs where one of entries contain an empty sentences.",
+        reference="https://github.com/sbintuitions/JMTEB",
+        dataset={
+            "path": "sbintuitions/JMTEB",
+            "name": "livedoor_news",
+            "revision": "e4af6c73182bebb41d94cb336846e5a452454ea7",
+            "trust_remote_code": True,
+        },
+        type="Clustering",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["jpn-Jpan"],
+        main_score="v_measure",
+        date=("2000-01-01", "2014-02-09"),
+        form=["written"],
+        domains=["News"],
+        task_subtypes=["Topic classification"],
+        license="cc-by-nd-2.1-jp",
+        socioeconomic_status="high",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="",
+        n_samples={"test": 1106},
+        avg_character_length={"test": 1082.61},
+    )
+
+    def dataset_transform(self):
+        self.dataset = self.dataset.rename_columns(
+            {"text": "sentences", "label": "labels"}
+        )
+
+        for split in self.metadata.eval_splits:
+            # remove empty sentences (there is only one pr. split)
```

Comment:
Is this what it stands for? Seems not needed to shorten one char with one char 😁
```suggestion
            # remove empty sentences (there is only one per split)
```